.gitignore

```diff
@@ -15,3 +15,4 @@ node-*
 /lib/coder-cloud-agent
 .home
 coverage
+code-server-*
```

Comment:
Added because if you detar the release package, you get code-server...and we don't want to accidentally commit that.

Reply:
We're missing a newline at the end of the file here!

Reply:
Maybe we should `cd release-packages && tar -xzf code-server*-linux-amd64.tar.gz` to keep the extraction inside the `release-packages` dir instead of in the root.

Reply:
Good idea! I will revert this and do that instead